added -t parameter to solve

exoji2e opened the pull request:
Increment on #12. Not sure how nice this is though? Thoughts @ahnlabb?

exoji2e committed:
added -t parameter to solve

ahnlabb commented:
I would prefer not making them options, instead set `nargs='?'`:
```
parser.add_argument(inp, nargs='?')
```
I believe the most common use case is use autocomplete in bash to find ans file (full path), "please find .in file for me" so assume that when creating defaults

exoji2e committed:
Handle if only one file is given to solve.py

solve.py tries to detirmine if one file is an
input or ans-file. If zero files are given as
arguments, solve.py will be run on every .ans-file
inside the submission directory.

exoji2e committed:
add shebang

exoji2e commented:
OK, what do you think about this?

exoji2e committed:
fix if args becomes None
